src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/CourseActivity.kt

```diff
@@ -89,8 +89,8 @@ data class CourseActivity(
   @Column
   val scheduleEndDate: LocalDate? = null,
 
-  @OneToMany(mappedBy = "courseActivity", cascade = [CascadeType.ALL], fetch = FetchType.LAZY)
-  var payRates: MutableList<CourseActivityPayRate> = mutableListOf(),
+  @OneToMany(mappedBy = "courseActivity", cascade = [CascadeType.ALL], fetch = FetchType.LAZY, orphanRemoval = true)
+  val payRates: MutableList<CourseActivityPayRate> = mutableListOf(),
```

Comment:
But for OneToMany prevent changing the list as this confuses Hibernate - must mutate the existing list instead.